spec/unit/package_spec.rb

```diff
@@ -22,5 +22,10 @@
       expect(chef_run).to install_hab_package("core/bundler")
         .with(version: "1.13.3/20161011123917")
     end
+
+    it "installs core/elasticsearch with a specific depot url" do
```

Comment:
your test description doesn't match the package install (elasticsearch vs hab-sup)